tests/test_remove.py

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env python3
+
+# Copyright 2019 The Kapitan Authors
+# SPDX-FileCopyrightText: 2020 The Kapitan Authors <kapitan-admins@googlegroups.com>
+#
+# SPDX-License-Identifier: Apache-2.0
+
+"remove tests"
+import filecmp
+import hashlib
+import os
+import shutil
+import sys
+import unittest
+
+from kapitan.cli import main
+from kapitan.inputs.copy import Copy
+from kapitan.inputs.remove import Remove
+from kapitan.utils import directory_hash
+
+search_path = ""
+ref_controller = ""
+test_path = os.path.join(os.path.dirname(os.path.abspath(__file__)), "test_remove")
+compile_path = os.path.join(test_path, "output")
+file_path = os.path.join(test_path, "input")
+test_file_path = os.path.join(file_path, "test_copy_input")
+test_file_compiled_path = os.path.join(compile_path, "test_copy_input")
+test_file_content = """
```

Comment:
It might be slightly cleaner to keep this yaml as a normal file next to other test resources?  For easier maintenance/updates to it?

Reply:
the file content doesn't actually matter here. It could easily just be one line. I lifted it from the `test_copy.py` file.

Reply:
Gotcha, no worries then. :)